raiden/tests/integration/api/test_pythonapi.py

```diff
@@ -2,7 +2,7 @@
 
 from raiden.api.python import RaidenAPI
 from raiden.tests.utils.transfer import get_channelstate
-from raiden.tests.utils.blockchain import wait_until_block
+from raiden.tests.utils.geth import wait_until_block
```

Comment:
I feel we should leave `wait_until_block` in `raiden.tests.utils.blockchain`, it's not related to geth.

Reply:
After discussion with Augusto we postponed this into another PR.